projects/plugins/jetpack/extensions/blocks/contact-form/editor.scss

```diff
@@ -468,3 +468,30 @@ input.components-text-control__input {
 .jetpack-contact-form__crm_toggle p {
 	margin-bottom: 0;
 }
+
+.jetpack-select {
+	display: inline-flex;
+	flex-direction: column;
+	min-width: 200px;
+
+	&__toggle {
+		border: var(--jetpack--contact-form--border-size) solid var(--jetpack--contact-form--primary-color);
```

Comment:
I guess we need to make sure this will still work if you choose to explicitly style your block. Not sure if that's already supposed to be part of the PR, just throwing it out there.

Reply:
When we explicitly style the block, the styles go to the `style` property, don't they?
If so, they would override this CSS.
Am I missing something?